### PR TITLE
add name to domain api

### PR DESCRIPTION
**Pull request by davidlumley**
We're looking at publicly releasing this next week, so I've gone ahead and added support to the Python package 👍

### CODE REVIEW

File: clearbit/__init__.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from clearbit.resource import Resource
 from reveal import Reveal
 from risk import Risk
+from name_to_domain import NameToDomain
```
Comment (maccman) [inline clearbit/__init__.py]: I think remove the underscores

Comment (davidlumley) [reply clearbit/__init__.py]: `name_to_domain` is the file name — are you running into errors running this or is there a convention I'm not aware of?

Comment (maccman) [reply clearbit/__init__.py]: Can you check, but I have a feeling that's the convention.

Comment (davidlumley) [reply clearbit/__init__.py]: I did check before I made the previous comment, but wasn't sure if you knew something I might not haha

Comment (warmwaffles) [reply clearbit/__init__.py]: I don't think it's against convention. Looking at PEP-8 and I don't see anything about it https://www.python.org/dev/peps/pep-0008/

**Pull request merged**